config/locales/fr.yml

```diff
@@ -843,6 +846,25 @@ fr:
           success: Samples were successfully transferred.
           error: "The following list of samples failed to transfer:"
       attachments:
+        concatenations:
+          create:
+            success: Files were successfully concatenated.
+            basename_help: The basename can only contain letters, numbers, hyphens, periods, and underscores
+          modal:
+            title: Concatenate Files
+            description: "The following files were selected to be concatenated:"
+            basename: Filename
+            delete_originals: Delete Originals
+            submit_button: Concatenate
+        deletions:
+          modal:
+            title: File Deletion
```

Comment:
Same comment above for the english translation file

Reply:
Fixed in 62fbc1ba10cbef28ca5e0f5bc9aee03f98487c49